www/js/app.js

```diff
@@ -1248,8 +1250,10 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             // We have to remove the focus from the search field,
             // so that the keyboard does not stay above the message
             $("#searchArticles").focus();
-            alert("Archive not set : please select an archive");
-            $("#btnConfigure").click();
+            uiUtil.systemAlert("Alert", "Archive not set : please select an archive", false)
+                .then(function () {
```

Comment:
I think again `.then....` can go on end of previous line, and then remove the _extra_ indenting.